src/constants.h

```diff
@@ -1,36 +1,41 @@
 #pragma once
 
 /**
- * Instruction class of the Boilerplate application.
+ * Instruction class of the Hathor application.
  */
 #define CLA 0xE0
 
 /**
- * Length of APPNAME variable in the Makefile.
+ * Maximum length of MAJOR_VERSION || MINOR_VERSION || PATCH_VERSION.
  */
-#define APPNAME_LEN (sizeof(APPNAME) - 1)
+#define APPVERSION_LEN 3
 
 /**
- * Maximum length of MAJOR_VERSION || MINOR_VERSION || PATCH_VERSION.
+ * Address length
  */
-#define APPVERSION_LEN 3
+#define ADDRESS_LEN 25
+
+/**
+ * B58 encoded address length
+ */
+#define B58_ADDRESS_LEN 35
 
 /**
- * Maximum length of application name.
+ * Token UID length
  */
-#define MAX_APPNAME_LEN 64
+#define TOKEN_UID_LEN 32
 
 /**
- * Maximum transaction length (bytes).
+ * Tx input length
  */
-#define MAX_TRANSACTION_LEN 510
+#define TX_INPUT_LEN 35
 
 /**
- * Maximum signature length (bytes).
+ * Pubkey hash length
  */
-#define MAX_DER_SIG_LEN 72
+#define PUBKEY_HASH_LEN 20
 
 /**
- * Exponent used to convert mBOL to BOL unit (N BOL = N * 10^3 mBOL).
+ * Maximum number of outputs on a transaction.
  */
-#define EXPONENT_SMALLEST_UNIT 3
+#define MAX_NUM_TX_OUTPUTS 100
```

Comment:
So we are limiting the max number of outputs in 100? We can't support 255 as our protocol max?

Reply:
Changed to 255